pkg/cli/cmd_run.go

```diff
@@ -86,6 +88,9 @@ func runRun(cmd *Command, rawArgs []string) error {
 	if runAutoRemove && runDetachFlag {
 		return fmt.Errorf("conflicting options: --detach and --rm")
 	}
+	if runGateway != "" && runIPAddress != "" {
+		return fmt.Errorf("conflicting options: --gateway and --ip-address")
+	}
```

Comment:
These are not conflicting options, you can configure a public ip address but still use a gateway to access its SSH port